lib/contract_interface/src/models.rs

```diff
@@ -87,7 +132,7 @@ pub struct CommitBatchInfo {
     pub priority_operations_hash: B256,
     pub dependency_roots_rolling_hash: B256,
     pub l2_to_l1_logs_root_hash: B256,
-    pub l2_da_validator: Address,
+    pub l2_da_commitment_scheme: DACommitmentScheme,
```

Comment:
If we don't get rid of batch persistence before this PR (cc @RomanBrodetski, iiuc it won't be done this week or so), then it should be done in a backward-compatible way -- l2_da_commitment_scheme should be optional

Reply:
Added default value, PTAL